cli/src/test/java/com/devonfw/tools/ide/context/GitContextTest.java

```diff
@@ -236,4 +236,65 @@ public void testIsRepositoryUpdateAvailableNoUpdates(@TempDir Path tempDir) {
     // assert
     assertThat(result).isFalse(); // No updates should be available
   }
+
+  /**
+   * Tests the conversion of a Git URL from HTTPS to SSH protocol.
+   * <p>
+   * Given a Git URL in HTTPS format, this test ensures that it is correctly converted to the SSH format using the
+   * {@link GitUrlSyntax#convertToPreferredProtocol(GitUrl, String)} method.
+   */
+  @Test
+  public void testConvertGitUrlFromHTTPSToSSH() {
```

Comment:
https://github.com/devonfw/IDEasy/blob/main/documentation/coding-conventions.adoc#naming
```suggestion
  public void testConvertGitUrlFromHttpsToSsh() {
```